cloud-consumption/src/main/resources/application.yml

```diff
@@ -116,11 +116,41 @@ cb:
   address.resolving.timeout: 60000
 
   cloud.api.executorservice.pool.size: 40
-
+  max:
```

Comment:
Sanity check: I suppose `cloud-consumption` would not explicitly depend on either of the configs below, and they are only included to satisfy the Spring config dependencies of the `cloud-*` modules (and avoid runtime errors when initializing the Spring `ApplicationContext`).

Reply:
yes actually it is the reason

Reply:
As @SubhanHakverdiyev also confirmed, you are right, that is the reason. I tried to minimize the added configs to the absolut necessary ones to avoid runtime errors.